src/drivers/ll40ls/LidarLiteI2C.cpp

```diff
@@ -126,7 +126,7 @@ int LidarLiteI2C::init()
 
 	_class_instance = register_class_devname(RANGE_FINDER_BASE_DEVICE_PATH);
 
-	if (_class_instance == CLASS_DEVICE_PRIMARY) {
+	if (_class_instance >= 0) {
```

Comment:
I can see that this was needed, to fix it, but now that multi orb advertisements and class instances are completely decoupled the publication should not depend on the class instance at all.